smoketests/tests/csharp_module.py

```diff
@@ -17,6 +17,7 @@ def test_build_csharp_module(self):
 
         try:
 
+            run_cmd("dotnet", "nuget", "locals", "all", "--clear", cwd=bindings, capture_stderr=True)
```

Comment:
clear package cache. this isn't required to make the CI work again, but there did seem to be some kind of caching happening because the smoketests weren't passing or failing consistently. This hopefully makes them more consistent.

Reply:
(we think this also resolves a hypothetical issue where multiple open PRs make non-backwards-compatible changes to the NuGet packages)